frontend/packages/console-shared/src/components/drawer/DrawerHeader.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { useSize, Size } from './drawer-utils';
+
+type DrawerHeaderProps = {
+  className?: string;
+  children: (size: Size) => React.ReactNode;
+};
+
+const DrawerHeader: React.FC<DrawerHeaderProps> = ({ className, children }) => {
```

Comment:
You should no longer need this component if you pass the header as a `prop` to `Drawer` and then render it inside a wrapper `div`.